src/helpers/version/mod.rs

```diff
@@ -46,7 +46,7 @@ pub async fn parse_version_type(client: &Client, version: &str) -> Result<Parsed
             } else if hash_regex.is_match(version) {
 
                 return Ok(ParsedVersion {
-                    tag_name: version.to_string().chars().take(7).collect(),
+                    tag_name: version.to_string(),
```

Comment:
I've encountered a problem: it seems like only full hash can be used. Otherwise, for example:
```
fatal: couldn't find remote ref 9cadbf1
```

Reply:
hmm fair enough, will see how bob will handle short hash for other things besides this

Reply:
Yes! Thank you. Somehow I've missed this field in the struct.

I've made the necessary changes.